tests/menu_items_registry.test.ts

```diff
@@ -357,6 +365,39 @@ describe("Menu Item actions", () => {
     });
   });
 
+  describe("Insert row above via row menu", () => {
+    const path = ["add_row_before"];
+
+    test("A selected row", () => {
+      selectRow(model, 4, "newAnchor");
+      expect(getName(path, env, rowMenuRegistry)).toBe("Insert row above");
```

Comment:
@hokolomopo I'll leave it for now because there are 139 places in this file the variable `path` occurring 😂 I don't feel it's a big issue for readability

Reply:
Maybe just change name `path` into somehting more meaningful would be better actually, like you did in your tests with `insertCellShiftDownPath`.

But yeah it's a detail :)